test/cli/repl/t10-avalanche/expected

```diff
@@ -6,91 +6,91 @@ ok, loaded test/cli/repl/data.psv, 13 rows
 ok, avalanche is now on
 > > -- A rather complicated feature to convert to Avalanche
 > - Avalanche:
-gen$date = DATE
+[32mgen[0m$[32mdate[0m = DATE
 {
-  init [Mutable] [Int] acc$c$conv$11 = 0 : Int;
-  init [Mutable] [(Sum Error (Buf Int))] acc$conv$22 = Right Buf 3 [] : (Sum Error (Buf Int));
-  load_resumable [Int] acc$c$conv$11;
-  load_resumable [(Sum Error (Buf Int))] acc$conv$22;
-  for_facts (gen$fact : ((Sum Error Int), DateTime)) in new {
-    read [Mutable] [(Sum Error (Buf Int))] acc$conv$22 = acc$conv$22;
-    write acc$conv$22 = Sum_fold# [Error] [Buf Int] [(Sum Error (Buf Int))]
+  init [Mutable] [Int] [32macc[0m$[32mc[0m$[32mconv[0m$[32m11[0m = 0 : Int;
+  init [Mutable] [(Sum Error (Buf Int))] [32macc[0m$[32mconv[0m$[32m22[0m = Right Buf 3 [] : (Sum Error (Buf Int));
+  load_resumable [Int] [32macc[0m$[32mc[0m$[32mconv[0m$[32m11[0m;
+  load_resumable [(Sum Error (Buf Int))] [32macc[0m$[32mconv[0m$[32m22[0m;
+  for_facts ([32mgen[0m$[32mfact[0m : ((Sum Error Int), DateTime)) in new {
+    read [Mutable] [(Sum Error (Buf Int))] [32macc[0m$[32mconv[0m$[32m22[0m = [32macc[0m$[32mconv[0m$[32m22[0m;
+    write [32macc[0m$[32mconv[0m$[32m22[0m = Sum_fold# [Error] [Buf Int] [(Sum Error (Buf Int))]
```

Comment:
Is there any way we can avoid this escape code stuff in the test output? It makes the diffs pretty hard to read